server/odc-migrate/src/main/resources/migrate/common/V_4_2_4_1__alter_notification_schema.sql

```diff
@@ -0,0 +1,104 @@
+DROP TABLE IF EXISTS `notification_channel`;
+CREATE TABLE `notification_channel` (
```

Comment:
do these tables not contain any unique keys?

Reply:
Thanks for your reminder, I checked the scripts again. In fact, the 'event_category' and 'event_name' fields in the 'notification_policy_metadata' table require unique constraints. I have added it.